setup.py

```diff
@@ -27,7 +27,6 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
-    ],
         "Programming Language :: Python :: 3.10",
```

Comment:
This looks like a merge/rebase conflict gone bad. `"Programming Language :: Python :: 3.10",` is here twice also.